node/resolvers/catalog/index.ts

```diff
@@ -106,6 +107,7 @@ const brandFromList = async (
   slug: string,
   catalog: Context['clients']['catalog']
 ) => {
+  // eslint-disable-next-line no-shadow
   const brandFromList = await getBrandFromSlug(toLower(slug), catalog)
 
   return brandFromList ? brandFromList.id : null
```

Comment:
`brandFromList` is the name of the function, so it cannot be used in the constant either.

[Playground](https://www.typescriptlang.org/play?#code/C4TwDgpgBAIghsOUC8UDeAoKUCWATAfgC4oBnYAJxwDsBzDAXwwwGMB7a8qPBOE+RCnRMMAMwCu1FsBwdc1HMAAUASnRYoFCMHEVq3XgQB0+KAQJRq4gDbXGzdpzbWIR621pKai1SqA)
```suggestion
  const data = await getBrandFromSlug(toLower(slug), catalog)

  return data?.id ?? null
```